src/bundles.ts

```diff
@@ -97,12 +101,13 @@ export async function uploadRemoteBundle({
   const apiParams = pick(options, ['baseURL', 'sessionToken', 'source', 'bundleHash']);
 
   const uploadFileChunks = async (bucketFiles: FileInfo[]): Promise<boolean> => {
-    const resp = await extendBundle({
+    let resp = await extendBundle({
```

Comment:
why are we changing this to `let`? I don't see re-assigning of the object 🤔

Reply:
just a typo. I needed to change it during some tests and then I forgot to revert it. Fixing it now